modular_skyrat/modules/blastwave_outfits/code/clothing/goner_head.dm

```diff
@@ -0,0 +1,78 @@
+/obj/item/clothing/head/goner
+	name = "blastwave helmet"
+	desc = "A plastic helmet with purple paint applied."
```

Comment:
Shouldn't say the color in there if it can be reskinned to another color.

Reply:
My bad, was tired at the moment I decided to redo everything.